src/components/vm/nics/vmNicsCard.tsx

```diff
@@ -664,16 +726,76 @@ export class VmNetworkTab extends React.Component<VmNetworkTabProps, VmNetworkTa
+
+            const Description = ({ term, children } : { term: string, children: React.ReactNode }) => (
+                <DescriptionListGroup>
+                    <DescriptionListTerm>{term}</DescriptionListTerm>
+                    <DescriptionListDescription>{children}</DescriptionListDescription>
+                </DescriptionListGroup>
+            );
+
+            const expandedContent = (
+                <DescriptionList isAutoFit>
+                    { this.state.haveAgentSource &&
+                        <Description term={_("MAC address")}>
+                            <div id={`${id}-network-${networkId}-mac`}>{target.mac}</div>
+                        </Description>
+                    }
+                    <Description term={_("Model type")}>
+                        <Flex
+                            spaceItems={{ default: 'spaceItemsSm' }}
+                            alignItems={{ default: 'alignItemsCenter' }}
+                            id={`${id}-network-${networkId}-model`}
+                        >
+                            <FlexItem>{target.model}</FlexItem>
+                            {needsShutdownIfaceModel(vm, target) &&
+                                <NeedsShutdownTooltip
+                                    iconId={`${id}-network-${networkId}-model-tooltip`}
+                                    tooltipId="tip-network"
+                                />
+                            }
+                        </Flex>
+                    </Description>
+                    <Description term={_("Type")}>
+                        <Flex spaceItems={{ default: 'spaceItemsSm' }} alignItems={{ default: 'alignItemsCenter' }} id={`${id}-network-${networkId}-type`}>
+                            <FlexItem>{target.type}</FlexItem>
+                            {needsShutdownIfaceType(vm, target) && <NeedsShutdownTooltip iconId={`${id}-network-${networkId}-type-tooltip`} tooltipId="tip-network" />}
+                        </Flex>
+                    </Description>
+                    <IPDescriptions
+                        ipInterfaces={this.state.ips.filter(ip => ip.mac === target.mac)}
+                        id={id}
+                        networkId={networkId}
+                    />
+                    <NetworkSourceDescriptions
+                        network={target}
+                        networkId={networkId}
+                        vm={vm}
+                        hostDevices={this.hostDevices}
+                    />
+                </DescriptionList>
+            );
+
+            const rowId = networkId;
+
             networkId++;
-            return { columns, props: { key: cockpit.format("$0-$1-$2", target.mac, target.address.bus || networkId, target.address.slot || '') } };
+            return {
+                columns,
+                props: {
+                    key: cockpit.format("$0-$1-$2", target.mac, target.address.bus || networkId, target.address.slot || ''),
```

Comment:
This added line is not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-2324-02ce13bf-20250922-141831-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)